lib/membrane_rtc_engine/media_event.ex

```diff
@@ -5,6 +5,10 @@ defmodule Membrane.RTC.Engine.MediaEvent do
   @type to_t() :: peer_id_t() | :broadcast
   @type sfu_media_event_t() :: {:sfu_media_event, to_t(), binary()}
 
+  @type media_event_type() ::
```

Comment:
To make it consistent with other type specs I would write 
```suggestion
  @type media_event_t() ::
```